<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>실시간 검색어 뷰어</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>130</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>130</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>검색어 예약</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LoopTimeEditor">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>160</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1000</string>
    </property>
   </widget>
   <widget class="QLabel" name="LoopTime">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>160</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>반복 시간 </string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>230</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StopButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>230</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>사이트</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="DaumWebSiteButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>60</y>
      <width>100</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>다음</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="LogEditor">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>270</y>
      <width>311</width>
      <height>191</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="PotalSearchList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>131</width>
      <height>411</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>1위</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2위</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3위</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4위</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5위</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>6위</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>7위</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>8위</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>9위</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>10위</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>검색어</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>By laskdjlaskdj12 all right reserved</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>20</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ver 1.0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>895</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
